packages/lib/README.md

```diff
@@ -72,3 +72,19 @@ various front-end development stacks:
 - [Vite](https://codesandbox.io/s/h5weblib-vite-xru04)
 - [Create React App v4](https://codesandbox.io/s/h5weblib-cra-v4-2te48)
 - [Create React App v5](https://codesandbox.io/s/h5weblib-cra-v5-nhznhh)
+
+### Browser support
+
+H5Web works out of the box on **Firefox 78 ESR**.
+
+Support for Firefox 68 ESR is possible by polyfilling the `ResizeObserver` API.
+One easy way to do this is with [polyfill.io](https://polyfill.io/v3/):
+
+```html
+<head>
+  <!-- title, meta, link, etc. -->
+  <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CResizeObserver"></script>
+</head>
+```
+
+Older versions of Firefox are not supported.
```

Comment:
Might be good to talk about other browsers ?

Well, we can keep it like this for now

Reply:
Agreed. I had the same line of thought when reading this section